nltk/twitter/twitter_demo.py

```diff
@@ -0,0 +1,231 @@
+# -*- coding: utf-8 -*-
+# Natural Language Toolkit: Twitter client
+#
+# Copyright (C) 2001-2015 NLTK Project
+# Author: Ewan Klein <ewan@inf.ed.ac.uk>
+# URL: <http://nltk.org/>
+# For license information, see LICENSE.TXT
+
+"""
+Examples to demo the :py:mod:`twitterclient` code.
```

Comment:
My reservations about migrating the contents of this file to a module docstring are:

* the demos don't work well as doctests in the ordinary sense since in most case their output is not predictable;
* the demos that 'follow' users in the live stream can take a few minutes to run, so it's good to have a way of selectively excluding them; it would be more complicated to do this in a docstring context;
* as noted, most of the demos won't work without the user first following the instructions in the `test/twitter.ipynb` notebook for obtaining and installing Twitter API keys, so I'd suggest that it would better to point the user to the notebook as an intro to the functionality of the code;
* from this point of view, the demo file could be removed altogether, although it's possibly a useful resource for some users.